db/upgrade-jambonz-db.js

```diff
@@ -159,6 +159,12 @@ const sql = {
     'CREATE INDEX client_sid_idx ON clients (client_sid)',
     'ALTER TABLE clients ADD CONSTRAINT account_sid_idxfk_13 FOREIGN KEY account_sid_idxfk_13 (account_sid) REFERENCES accounts (account_sid)',
     'ALTER TABLE sip_gateways ADD COLUMN protocol ENUM(\'udp\',\'tcp\',\'tls\', \'tls/srtp\') DEFAULT \'udp\''
+  ],
+  8005: [
```

Comment:
check below in the code...further down.  I think there is a series of 'if' statements you'll need to add to

Reply:
omg missed this. :D

fixed